core/src/main/scala/org/apache/spark/sql/rapids/tool/util/EventUtils.scala

```diff
@@ -275,4 +281,15 @@ object EventUtils extends Logging {
         None
     }
   }
+
+  private def processSparkRapidsBuildInfo(event: String) {
+    implicit val formats: Formats = DefaultFormats
+    val jsonMap = parse(event).extract[Map[String, Any]]
+    val sparkRapidsInfo = jsonMap("sparkRapidsBuildInfo").asInstanceOf[Map[String, String]]
+    ToolUtils.sparkRapidsRuntimeVersion = Some(sparkRapidsInfo("version"))
```

Comment:
any reason not to just dump everything in the maps out?

Reply:
I'm not sure specific requirements here as its vague from issue... if we are dumping them in runtime you could do something like you have with sparkRapidsBuildInfo.$key=$value
If someone is trying to figure out exact code it matches, There are definitely other fields that are important, like the build revision, user, etc (https://github.com/NVIDIA/spark-rapids/pull/10752)

Reply:
Thanks @tgravescs! I updated the PR description for the latest changes.